learn/core_concepts/indexes.md

```diff
@@ -48,55 +35,91 @@ The `uid` is set at [index creation time](/reference/api/indexes.md#create-an-in
 
 [Learn more about synonyms](/learn/configuration/synonyms.md)
 
-## Stop words
+### Filterable attributes
+
+Filtering allows you to refine your search based on different categories, in our case, document attributes; these can include a movie genre, a price range, or location. To filter by any document attribute, you need to add it to `filterableAttributes` using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update filterable attributes endpoint](/reference/api/filterable_attributes.md#update-filterable-attributes).
+
+Adding an attribute to `filterableAttributes` will not impact search. You need it to use the [`filter` search parameter](/reference/api/search.md#filter) to refine your results.
+
+[Learn more about filtering](/learn/advanced/filtering_and_faceted_search.md)
+
+### Sortable attributes
 
-Sometimes you may want to ignore certain words in documents and search queries. To do so, **a set of stop words can be defined for an index**. Unless you actually need them, some words neither add semantic value nor context. Besides, they are often too frequent (for example, `the` or `of` in English).
+By default, Meilisearch orders results according to their relevancy. You can alter this sorting behavior to show certain results first by adding attributes to the `sortableAttributes` array using the [update settings endpoint](/reference/api/settings.md#update-settings) or the [update sortable attributes endpoint](/reference/api/sortable_attributes.md#update-sortable-attributes). You can then use the [`sort` search parameter](/reference/api/search.md#sort) to sort your results in ascending or descending oder.
 
-Words added to the [stop words list](/reference/api/stop_words.md) will be ignored during search. In addition to improving relevancy, designating common words as stop words also greatly improves performance.
+[Learn more about sorting](/learn/advanced/sorting.md)
 
-For example, suppose you want to search for `the great gatsby`. You would prefer to receive documents containing the terms `great gatsby`, rather than documents containing the terms `the great`, or just `the`. In this case, adding `the` to the stop word list would improve performance and make search results more relevant.
+### Stop words
+
+Your dataset may contain words you want to ignore during search. These may include words that don't add semantic value or context or are too frequent (e.g., `the` or `of` in English). You can add such words to the [stop words list](/reference/api/stop_words.md) and Meilisearch will ignore them during search. In addition to improving relevancy, designating common words as stop words also greatly improves performance.
```

Comment:
```suggestion
Your dataset may contain words you want to ignore during search because, for example, they don't add semantic value or occur too frequently (e.g., `the` or `of` in English). You can add these words to the [stop words list](/reference/api/stop_words.md) and Meilisearch will ignore them during search.
```